docs/data/destinations/meta-pixel.md

```diff
@@ -3,16 +3,18 @@ title: Meta Pixel Event Streaming
 description: Amplitude Data's Meta Pixel integration lets you stream your Amplitude event data straight to Meta Pixel with just a few clicks.
 ---
 
-Amplitude Data's Meta Pixel integration lets you stream your Amplitude event data straight to Meta Pixel with just a few clicks.
-
 --8<-- "includes/open-beta.md"
 
+Amplitude Data's Meta Pixel integration lets you stream your Amplitude event data straight to Meta Pixel with just a few clicks.
+
 ## Considerations
 
-Keep these things in mind when sending Amplitude data to Meta. 
+Keep these things in mind when sending Amplitude data to Meta.
 
-- Amplitude uses the `user_id` to match the `external_id` user property field in Meta Pixel. Make sure that this field is present in your Amplitude events.
-- All event properties that Amplitude sends are stored as 'custom data'. Because of this, normal Meta conversion can't be applied to those events. You must create a custom conversion using this [Meta guide](https://www.facebook.com/business/help/2375212726097833?id=1205376682832142 "https://www.facebook.com/business/help/2375212726097833?id=1205376682832142").
+- Amplitude uses the `external_id` user property field in Meta Pixel for user matching.
+- Amplitude allows you to customize mappings that will appear in the `user_data` array of Meta Pixel for conversion.
```

Comment:
What does this show up as in the Meta Pixel UI? That information might be useful, rather than just the API field we send it in.

I also never reviewed Meta Pixel so not sure, but is this where we send properties too?

Reply:
No this was different from custom fields in SPF, but I can clarify this more. The `user_data` array is not necessarily a UI thing but it's Meta Pixel specified fields for better customer matching